src/cache_cf.cc

```diff
@@ -3786,40 +3813,45 @@ parsePortCfg(AnyP::PortCfgPointer *head, const char *optionName)
 
     s->secure.syncCaFiles();
 
+    const auto &rawFlags = s->flags.rawConfig();
     if (s->transport.protocol == AnyP::PROTO_HTTPS) {
         s->secure.encryptTransport = true;
 #if USE_OPENSSL
         /* ssl-bump on https_port configuration requires either tproxy or intercept, and vice versa */
-        const bool hijacked = s->flags.isIntercepted();
-        if (s->flags.tunnelSslBumping && !hijacked) {
+        if (rawFlags.tunnelSslBumping && !s->flags.interceptedSomewhere()) {
```

Comment:
I disagree.
* in the current PR these two lines are equivalent, since TrafficMode::proxySurrogateHttpsSslBump() returns false.
* in the upcoming PR where TrafficMode::proxySurrogateHttpsSslBump() is implemented we would just need to adjust the message below. The new condition is correct, verifying the new case when tunnelSslBumping is supported (i.e., PROXY+https_port).